monkeys/demanding-monkey/README.md

```diff
@@ -0,0 +1,12 @@
+# Chaos Test: Demanding Monkey 
+
+![](https://github.com/cilium/chaos-monkeys/raw/master/monkeys/demanding-monkey/.img/impatience.jpg)
+
+## Test Description
+
+Continuously queries a URL and the Kubernetes API server. If requests fail or take above a certain amount of time,
+will fail.
+
+## Configuration
+
+* `WATERMARK`: if a request takes above this amount of time (in seconds), will be considered a failure. 
```

Comment:
Would be great to be specific and say that this relates to `--max-time` of curl. It would also be great to have a variable for `--connect-timeout` so we adjust it easily.

Reply:
Done.